congest_gh_config.yaml

```diff
@@ -0,0 +1,52 @@
+graphhopper:
+  graph.location: transit_data/graphhopper
+  routing.ch.disabling_allowed: true
+  routing.max_visited_nodes: 1500000
+
+  # graph.bytes_for_flags: 112
+
+  # Uncomment this if the export was done with turn-restriction-aware contraction hierarchies
+  # prepare.ch.edge_based: edge_and_node
+
+  #  datareader.file: oregon-latest.osm.pbf
+  #  graph.flag_encoders: car|turn_costs=true
+  #  graph.bytes_for_flags: 64
+  #
+  #  prepare.ch.weightings: fastest
+
+  profiles:
```

Comment:
Are these profiles documented somewhere? I haven't looked over everything. would it make more sense to name by start/end hour?

Reply:
Good questions - these are basically the "new" way of defining vehicles/"flag encoders" in GH (we used to just have a list of vehicle flag encoders here - `car`, `bike`, etc - but now you define profiles that include both vehicle and CH weighting, all at once). 

Because of this, they aren't really documented anywhere (unless you count the comment in the client code that lays out the time chunks each profile corresponds to). Right now, they're named with the actual hour of congestion the profile represents (so, `fastest_car_2` holds speeds from hour `2` of the synthetic speeds file that was loaded w/ the custom speed export DM). They can definitely be changed if it would improve clarity, though!

Reply:
got it. I misunderstood and thought it was a range of time, but that makes sense. what about "best" vs. "fastest"? Want to add a comment about it above? we can also put some information in the python repo in a different pr.

Reply:
Explained the best/fastest distinction a bit more here - https://github.com/replicahq/model/pull/2830#discussion_r453818637

Perhaps just removing these prefixes (`best_` and `fastest_`) in all instances (here, the client code in the python repo, and the R5 export script code) would make things clearer. Either way, though, I'll add some comments both here and in the client code in https://github.com/replicahq/model/pull/2830 to explain the different profiles a bit more clearly.

Reply:
Yes! In favor of eventually removing the prefixes. This is my GH-legacy-influenced terminology, I haven't fully transitioned from vehicle+weighting to profile yet.

Reply:
@kaelgreco @michaz went ahead and removed all of the prefixes from all 3 related PRs (this, https://github.com/replicahq/model-r5/pull/100, and https://github.com/replicahq/model/pull/2830), and added a comment where we create the unique bike weighting [here](https://github.com/replicahq/model-r5/pull/100/files#diff-d2c9d724b808b8a91479a3dc66a05d44R67-R74) so we remember that this important distinction exists. I think it's probably worth getting rid of this debt now rather than later on, just to simplify things.